packages/host/app/components/operator-mode/container.gts

```diff
@@ -319,12 +304,38 @@ export default class OperatorModeContainer extends Component<Signature> {
         here.addToStack(newItem);
         return await newItem.request?.promise;
       },
-      viewCard: async (card: Card, format: Format = 'isolated') => {
-        let itemsCount = here.stacks[stackIndex].length;
-
-        let currentCardOnStack = here.getCard(
-          here.stacks[stackIndex][itemsCount - 1]!
-        ); // Last item on the stack
+      viewCard: async (
+        card: Card,
+        format: Format = 'isolated',
+        fieldType?: 'linksTo' | 'contains' | 'containsMany' | 'linksToMany',
+        fieldName?: string
+      ) => {
+        let stack = here.stacks[stackIndex];
+        let itemsCount = stack.length;
+
+        let currentCardOnStack = here.getCard(stack[itemsCount - 1]!); // Last item on the stack
+
+        // TODO this is a hack until contained cards go away
```

Comment:
is there somewhere to record this so it can be cleaned up when needed?

Reply:
My hope is that the type change necessary will reveal all these places that need to update. Specially if the stack item type no longer includes `contained`, that should be enough for us to find all these items.